vars/transportRequestCreate.groovy

```diff
@@ -23,7 +23,8 @@ def call(parameters = [:]) {
 
     handlePipelineStepErrors (stepName: STEP_NAME, stepParameters: parameters) {
 
-        def script = parameters?.script ?: [commonPipelineEnvironment: commonPipelineEnvironment]
+        def script = parameters?.script
+        if(! script) error "[ERROR][${STEP_NAME}] No reference to surrounding script provided with key 'script', e.g. 'script: this'."
```

Comment:
To avoid code duplication and increase maintainability I would recommend to extract these lines into a common methods/step.

Reply:
could be handled in the `handlePipelineStepErrors` step..

Reply:
> could be handled in the handlePipelineStepErrors step..

@CCFenner : sound cool, but does not work with the current code base since e.g. `prepareDefaultValues` is called from `ConfigurationHelper.initDefaults`. Here no parameter map (and with that no `script`) is provided. Inside `prepareDefaultValues` there is  - of course - a call to `handlePipelineStepErrors` and there we fail with the script check.

Maybe it would be a good pattern to have dependencies only from steps (contained in `vars`) to classes inside `src` but no dependencies in the other direction.

Reply:
> To avoid code duplication 

@daniel-kurzynski : done, we have now a Prerequisite class performing a check for the availability of the script.